server/config.ts

```diff
@@ -222,7 +222,15 @@ export default {
       roles: [Roles.ACTIVITY_HUB],
     },
     {
-      route: '/activities/:mode(create|edit|view)',
```

Comment:
No regex is permitted in routes now
https://expressjs.com/en/guide/migrating-5.html#path-syntax